frontend/public/components/monitoring.tsx

```diff
@@ -1834,31 +1836,37 @@ type Silences = {
   loadError?: string;
 };
 
-type PrometheusAlert = {
+export type PrometheusAlert = {
   activeAt?: string;
   annotations: PrometheusLabels;
   labels: PrometheusLabels & {
     alertname: string;
   };
-  state: AlertStates;
-  value?: number;
+  state: AlertStates | string;
```

Comment:
Shouldn't this stay as just `state: AlertStates`? I think `AlertStates` should be a complete list of possible values for this field.

Reply:
Thanos rules states are capitalized.